packages/swap/src/routes/__tests__/v2/swap.test.ts

```diff
@@ -2271,5 +2275,107 @@ describe('server', () => {
         chunkIntervalBlocks: 3,
       });
     });
+
+    it(`returns the vault swap details in ${StateV2.Receiving} status`, async () => {
+      const txId = '0x1234';
+      env.CHAINFLIP_NETWORK = 'mainnet';
+      vi.mocked(getVaultSwapDetails).mockResolvedValue({
+        txId,
+        affiliateFees: [
+          {
+            account: 'cFHtoB6DrnqUVY4DwMHCVCtgCLsiHvv98oGw8k66tazF2ToFv',
+            bps: 10,
+          },
+        ],
+        amount: 100n,
+        brokerFee: {
+          account: 'cFHsUq1uK5opJudRDczhdPVj6LGoVTqYsfj71tbHfKsTAzkJJ',
+          bps: 10,
+        },
+        ccmDepositMetadata: {
+          channelMetadata: {
+            ccmAdditionalData: '4d4f5245',
+            gasBudget: '0x3039',
+            message: '48454c4c4f',
+          },
+          sourceAddress: {
+            Eth: '0xcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcf',
+          },
+          sourceChain: 'Ethereum',
+        },
+        dcaParams: {
+          chunkInterval: 100,
+          numberOfChunks: 5,
+        },
+        depositChainBlockHeight: 1,
+        destinationAddress: '0xcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcfcf',
+        inputAsset: {
+          asset: 'ETH',
+          chain: 'Ethereum',
+        },
+        maxBoostFee: 5,
+        outputAsset: {
+          asset: 'FLIP',
+          chain: 'Ethereum',
+        },
+        refundParams: {
+          minPrice: 0n,
+          refundAddress: '0x541f563237a309b3a61e33bdf07a8930bdba8d99',
+          retryDuration: 3,
+        },
+      });
+      const { body, status } = await request(server).get(`/v2/swaps/${txId}`);
+
+      expect(status).toBe(200);
+      expect(body.state).toBe('RECEIVING');
+      expect(body).toMatchInlineSnapshot(`
+        {
+          "boost": {
+            "maxBoostFeeBps": 5,
+          },
+          "brokers": [
+            {
+              "account": "cFHsUq1uK5opJudRDczhdPVj6LGoVTqYsfj71tbHfKsTAzkJJ",
+              "bps": 10,
+            },
+            {
+              "account": "cFHtoB6DrnqUVY4DwMHCVCtgCLsiHvv98oGw8k66tazF2ToFv",
+              "bps": 10,
+            },
+          ],
+          "ccmParams": {
+            "ccmAdditionalData": "4d4f5245",
```

Comment:
might confuse integrators if this is only here until the swap happens, can we remove it?